### ISSUE TITLE
Audit our current documentation with regard to accessibility (Where do we fail ? Where are we doing good?)

### ISSUE THREAD

**Issue by Spredzy** [ansible-community/community-team#42]
**Where are we currently at with accessibility ?**

Goal of this issue is to audit our current accessibility forces and gaps. What is it we are doing good and we should keep doing as well as what can we improve on and leverage #41 findings to implement

**Expected Results**

The description of the current state of our documentation accessibility wise. The good and the bad. So we have a clear idea of where we are at and can proceed with improvement.
